sharedbuffers/pool.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import tempfile
+import mmap
+import struct
+
+from .mapped_struct import StrongIdMap
+
+# Default section size is set to 128MB which is a size at which most
+# malloc implementations turn to mmap
+DEFAULT_SECTION_SIZE = 128<<20
```

Comment:
According to __man mallopt__, this threshold is at 128**KB**. On the other hand, it appears such threshold is dynamic, although it can be controlled with ```mallopt```.

Reply:
It's dynamic, and the top limit for how big that dynamic threshold can get is 64MB.

Reply:
By default, of course